README.md

```diff
@@ -1,199 +1,199 @@
+- **Aggregators** uniting decentralized protocols and liquidity sources.
 
-**Technical Showcase**
+We recommend using [Sol2ink](https://github.com/Brushfam/sol2ink) to accelerate development. Learn more in the [Sol2ink docs](https://learn.brushfam.io/docs/sol2ink).
 
-This category involes the development of applications which demonstrate the advantages of ink! over other smart contract languages (e.g. Solidity).
+### Technical Showcase
 
-The case doesn't necessarily need to be business oriented, but it should showcase ink!'s unique strengths.
+This category involves applications demonstrating ink!'s advantages over other smart contract languages, such as:
 
-You can use following aspects as indicators:
-- efficient gas usage
-- decreased smart contract size
-- improved developer experience
-- improved debugging experience
+- Efficient gas usage
+- Decreased contract size
+- Improved developer and debugging experience
 
-**Infrastructure**
+### Infrastructure
 
-The goal of this track is to improve the ink! ecosystem's developer infrastructure and tooling. We welcome you to be innovative and submit your ideas. We will prefer ideas that add differentiation to the Polkadot/ink! ecosystem, rather than tools that simply duplicate EVM.
+This category focuses on improving the ink! ecosystem's developer infrastructure and tooling. Innovative ideas that add differentiation to Polkadot/ink! are preferred over duplications of EVM tools.
 
 Examples of successful infrastructure products:
+
 - [OpenBrush](https://openbrush.io/)
 - [Swanky](https://github.com/AstarNetwork/swanky-cli)
-- Oracles (price feed, VRF, customizable oracles, ...)
-- Automation & account abstraction
-- DAO toolings (smart contract implementations, UI, SDKs)
-- Bridges (XCM integration, and bridges to external blockchains)
+- Oracles (price feeds, VRF, customizable oracles, etc.)
+- DAO tooling (smart contracts, SDKs, UIs)
+- Bridges (XCM integration, external blockchain bridges)
 
-### Bounty Levels
-The Smart Contract Bounty Program offers different bounty levels to help you best depending on your current stage.
+---
+
+**Bounty Levels**
 
-#### :hatching_chick: Level 1
+The Smart Contract Bounty Program offers different levels based on your stage:
+
+### 🐣 Level 1
 
 - **Target:** Individuals & small teams
 - **Amount:** Up to $10,000
 - **Requirements:** 4 approvals
 
-#### :baby_chick: Level 2
+### 🐤 Level 2
 
-- **Target:** Small teams/start-ups
+- **Target:** Small teams/startups
 - **Amount:** Up to $30,000
 - **Requirements:** 4 approvals
 
-#### :rooster: Level 3
+### 🐓 Level 3
 
-- **Target:** Companies/foundations with a proven track record
+- **Target:** Companies/foundations with proven track records
 - **Amount:** Up to $250,000
-- **Requirements:** 5 approvals (for >$100k: Smart Contract Bounty Council approval + Pitch call by request)
+- **Requirements:** 5 approvals (for >$100k: Smart Contract Bounty Council approval + optional pitch call)
```

Comment:
Who is the Smart Contract Bounty Council?

Reply:
The curators are